src/js/components/List/List.js

```diff
@@ -504,6 +505,24 @@ const List = React.forwardRef(
                     {content}
                     {orderControls}
                   </StyledItem>
+                ) : (
+                  <StyledItem
+                    key={key}
+                    tag="li"
+                    flex={false}
+                    pad={pad || theme.list.item.pad}
+                    background={adjustedBackground}
+                    border={adjustedBorder}
+                    {...boxProps}
+                    {...clickProps}
+                    {...orderProps}
+                  >
+                    <Box direction="row" align={orderAlign} flex {...boxProps}>
```

Comment:
Won't `{...boxProps}` always overwrite what is specified in `direction` and `align`? Should it precede these props?

Reply:
Your right changing now